specification/communication/data-plane/Microsoft.CommunicationServicesPhoneNumbers/stable/2021-03-07/examples/GetPhoneNumber.json

```diff
@@ -0,0 +1,31 @@
+{
+  "parameters": {
+    "api-version": "2021-03-07",
+    "endpoint": "https://resourcename.communication.azure.com/",
+    "phoneNumber": "+11234567890"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "id": "11234567890",
+        "phoneNumber": "+11234567890",
+        "countryCode": "US",
+        "assignmentType": "user",
+        "phoneNumberType": "geographic",
+        "capabilities": {
+          "calling": "inbound+outbound",
+          "sms": "inbound"
+        },
+        "purchaseDate": "2020-11-19T16:31:49.048Z",
+        "callbackUri": "callback.net",
```

Comment:
We should remove `callbackUri` and `applicationId`, right?

Reply:
Good catch.